app/src/main/java/com/example/weatherapp/models/WeatherDetailsUiState.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.weatherapp.models
+
+import retrofit2.Retrofit
+
+data class WeatherDetailsUiState(
+    val isPageLoaded: Boolean = false,
+    val weatherInformation: LocationWeatherData? = null,
+    val weatherRetroFit: Retrofit? = null,
```

Comment:
is `weatherRetroFit` being used somewhere?

Reply:
Completely forgot about this! I don't believe it is in use
That would mean that my View would know about my data layer and be able to contact the data layer which is what I don't want

Reply:
Removed it